src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/SelectResultSet.java

```diff
@@ -289,26 +303,26 @@ private String getFieldName(Field field) {
      * will be used.
      */
     private List<Schema.Column> populateColumns(Query query, String[] fieldNames, Map<String, FieldMappingMetaData> typeMappings) {
-        List<String> fields;
+        List<String> fieldNameList;
 
         if (isSelectAll() || containsWildcard(query)) {
-            fields = new ArrayList<>(typeMappings.keySet());
+            fieldNameList = new ArrayList<>(typeMappings.keySet());
         } else {
-            fields = Arrays.asList(fieldNames);
+            fieldNameList = Arrays.asList(fieldNames);
         }
 
         /*
-         * The reason the 'fieldMap' mapping is needed on top of 'fields' is because the map would be empty in cases
-         * like 'SELECT *' but List<String> fields will always be set in either case. That way, 'fields' is used to
+         * The reason the 'fieldMap' mapping is needed on top of 'fieldNameList' is because the map would be empty in cases
+         * like 'SELECT *' but List<String> fieldNameList will always be set in either case. That way, 'fieldNameList' is used to
          * access field names in order that they were selected, if given, and then 'fieldMap' is used to access the
          * respective Field object to check for aliases.
          */
         Map<String, Field> fieldMap = fetchFieldMap(query);
         List<Schema.Column> columns = new ArrayList<>();
-        for (String field : fields) {
+        for (String fieldName : fieldNameList) {
             // _score is a special case since it is not included in typeMappings, so it is checked for here
-            if (field.equals("_score")) {
-                columns.add(new Schema.Column(field, fetchAlias(field, fieldMap), Schema.Type.FLOAT));
+            if (fieldName.equals("_score")) {
```

Comment:
that _score string repeats twice, consider extracting to named constant

Reply:
I was thinking cover it in new `FieldMapping` class. I feel it's not that common, so finally I only add a `isMetaField`. Will add constant for it.

Reply:
I found that there are quite a few more in the whole codebase. For the time being, I'm just adding a constant for the 2 occurrences you mentioned in `SelectResultSet`. (`Field` is domain object for SQL domain area. I think we should move this constant to some new class in our `esdomain` package).